pkg/agent/proxy/proxier.go

```diff
@@ -978,6 +1010,10 @@ func NewProxier(
 		p.endpointsConfig = config.NewEndpointsConfig(informerFactory.Core().V1().Endpoints(), resyncPeriod)
 		p.endpointsConfig.RegisterEventHandler(p)
 	}
+	if p.multiclusterEnabled {
+		// serviceCIDRIPv4 is only required when Antrea Multi-cluster is enabled.
```

Comment:
How about "serviceCIDRIPv4 is required only for Multi-cluster Services"?

Reply:
Done